### PR TITLE
Change return type of penguin_bravo to short

### PR DESCRIPTION
**Pull request by t-animal**
This conforms with the second exercise and will be used to show
CI pipeline failing.

### CODE REVIEW

File: penguin/bravo.c [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * to something other than zero.
  */
 
-int penguin_bravo(void) {
+short penguin_bravo(void) {
```
Comment (dnnr) [inline penguin/bravo.c:8]: I really like this change, using `short` is more appropriate. However, you seem to have forgotten to update the declaration in the `interfaces.h` file as well, and that's why the build fails.

File: penguin/bravo.c [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
    return 0;
```
Comment (dnnr) [inline penguin/bravo.c:9]: I thought the plan was to change the return value here as well? But I guess the order doesn't matter, just keep it in mind for a later PR.

**Review by dnnr — CHANGES_REQUESTED** [review]
Looks almost good to me, just fix the build failure and then we're good to go.

File: penguin/bravo.c (hunk 2 above)
Comment (t-animal) [reply penguin/bravo.c:9]: Yes, thanks for reminding me, there is another PR open for this.

File: penguin/bravo.c (hunk 1 above)
Comment (t-animal) [reply penguin/bravo.c:8]: I have fixed the interface. Can you please re-check.

Comment (dnnr) [reply penguin/bravo.c:8]: Looks good, thanks!

**Review by dnnr — APPROVED** [review]

**Pull request merged**